examples/neural_style_transfer.py

```diff
@@ -50,8 +50,7 @@
 '''
 
 from __future__ import print_function
-from keras.preprocessing.image import load_img, img_to_array
-from scipy.misc import imsave
+from keras.preprocessing.image import load_img, save_img, img_to_array
```

Comment:
img_to_array is not used

Reply:
In this script `img_to_array` is used in the `preprocess_image` method here:
https://github.com/keras-team/keras/blob/b09ec1c9bf827622a5eebb94ba0af5bf202ca359/examples/neural_style_transfer.py#L98-L103